src/main/java/org/jitsi/impl/protocol/xmpp/ChatMemberImpl.java

```diff
@@ -297,12 +297,12 @@ void processPresence(Presence presence)
         }
 
         boolean wasVideoMuted = isVideoMuted;
-        StandardExtensionElement videoMutedExt = presence.getExtension("videomuted", "jabber:client");
-        isVideoMuted = videoMutedExt == null || Boolean.valueOf(videoMutedExt.getText()); /* defaults to true */
+        VideoMutedExtension videoMutedExt = presence.getExtension(VideoMutedExtension.class);
```

Comment:
I think this is incorrect. The `VideoMutedExtension` class uses a custom name space (for whatever reason), while the jitsi-meet client does not bother adding a namespace and it inherits "jabber:client".

Reply:
I'm tempted to update jitsi-xmpp-extensions to set `org.jitsi.xmpp.extensions.jitsimeet.VideoMutedExtension#NAMESPACE` to `jabber:client`. The element isn't used anywhere before the above change.
WDYT?

Reply:
if namespaces will be changed, be careful for breaking older mobile clients if they would not be sending the new one and if jicofo could ignore it?

Reply:
oh I didn't read the whole thing at first, Boris wrote it's behind a feature flag? then should be fine

Reply:
@paweldomas see jitsi/jitsi-xmpp-extensions#56. The `VideoMutedExtension` was never used - unless the GitHub code search is tricking me.